### PR TITLE
Convert views/activations Slim templates to ERB

### PR DESCRIPTION
**Pull request by spavlishak**

### CODE REVIEW

**Comment by rhombl4** [comment]
What is the reason for converting slim to erb?

File: app/views/activations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div id="tips-wrapper">
+  <div id="tips">
+    <p>
+      <span><%= t('.tip', email: current_user.email) %></span>
```
Comment (yivo) [inline app/views/activations/new.html.erb]: @spavlishak this output should not be escaped

File: app/views/activations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    </p>
+    <p>
+      <span><strong class='text-danger'><%= t('.spam') %></strong></span>
```
Comment (yivo) [inline app/views/activations/new.html.erb]: @spavlishak you should always use double quotes for HTML attributes

**Comment by dmk** [comment]
@rhombl4 simply because we are more comfortable with erb and we're going to change many pages.

File: app/views/activations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    </p>
+  </div>
+</div>
+<div class="resend-button">
+  <%= link_to activation_path(@activation), method: :put, class: 'btn btn-primary first' do %>
+    <span class="fui-mail"></span>
+    <span><%= t('helpers.submit.activation.update') %> </span>
```
Comment (yivo) [inline app/views/activations/new.html.erb]: @spavlishak extra space detected at the end

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
@spavlishak Could you please squash commits and write good message (changelog):

`Convert views/activations Slim templates to ERB`

**Review by yivo — APPROVED** [review]

**Pull request merged**